frontend/index.html

```diff
@@ -1,10 +1,39 @@
 <!doctype html>
 <html lang=en>
   <head>
-    <meta charset=utf-8>
```

Comment:
to musi być w pierwszych 1024 bajtach pliku, i ja dla świętego spokoju zostawiam to w pierwszej linijce head jakby komuś na dynamicznej stronie wpadł pomysł 1000+ znaków w tytule